### PR TITLE
Navigation drawer and romanian translation update

### PR DESCRIPTION
**Pull request by danielFerecatu**

### CODE REVIEW

**Comment by EzraBrooks** [comment]
Hi, can you provide a description of the changes in this pull request? Why did you change the height of the navigation drawer from the default height resource to 144dp?

File: app/res/layout/fragment_navigation_drawer.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="@dimen/abc_action_bar_default_height_material"
+        android:layout_height="144dp"
```
Comment (fadils) [inline app/res/layout/fragment_navigation_drawer.xml]: Why change?

File: app/res/layout/fragment_navigation_drawer.xml [hunk 2]
```diff
@@ -17,23 +17,24 @@
...
         <ImageView
-            android:layout_width="32dp"
-            android:layout_height="32dp"
+            android:layout_width="56dp"
+            android:layout_height="56dp"
             android:id="@+id/user_picture"
-            android:layout_centerVertical="true"
-            android:layout_margin="16dp" />
+            android:layout_margin="16dp"
+            android:layout_marginTop="16dp" />
```
Comment (fadils) [inline app/res/layout/fragment_navigation_drawer.xml]: Why declare a `marginTop` after `margin` is defined with the same 16dp?

File: app/res/layout/fragment_navigation_drawer.xml [hunk 3]
```diff
@@ -17,23 +17,24 @@
...
 
         <TextView
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="56dp"
```
Comment (fadils) [inline app/res/layout/fragment_navigation_drawer.xml]: I think, `wrap_content` is better. Should keep it.

File: app/res/layout/fragment_navigation_drawer.xml [hunk 4]
```diff
@@ -17,23 +17,24 @@
...
             android:textColor="#DEDEDE"
             android:layout_marginEnd="16dp"
             android:layout_marginRight="16dp"
-            android:layout_centerVertical="true"
-            android:layout_toRightOf="@+id/user_picture"
-            android:layout_toEndOf="@id/user_picture" />
+            android:layout_below="@+id/user_picture"
+            android:layout_marginLeft="16dp"
+            android:textSize="14sp"
+            android:textStyle="bold" />
```
Comment (fadils) [inline app/res/layout/fragment_navigation_drawer.xml]: should keep the previous.

File: app/res/layout/navigation_drawer_list_item_text.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <com.github.mobile.ui.CheckableRelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="48dp"
```
Comment (fadils) [inline app/res/layout/navigation_drawer_list_item_text.xml]: Same here. Why not `wrap_content`?

**Comment by fadils** [comment]
Also, this should be separated into two PRs, instead of one.

**Comment by danielFerecatu** [comment]
@fadils i refered to [http://www.google.com/design/spec/patterns/navigation-drawer.html](http://www.google.com/design/spec/patterns/navigation-drawer.html), followed the specs and come up with this, as you told me in [#693](https://github.com/github/android/issues/693). What i did wrong?

**Comment by fadils** [comment]
@danielFerecatu why close the PR? You can answer some of my questions above (and Ezra's), then revise the PR.
We all want good, clean code and clear git history. Going back and forth during PRs is pretty usual and you shouldn't be discouraged by it..

**Comment by danielFerecatu** [comment]
@fadils i'm not discouraged by it but all this thing with github is new for me and i barely know how things work, i don't have a workflow in Android Studio and beside that i don't even know Java programing. I wanted to contribute somehow but this project is far bigger than i thought and realised that this isn't appropriate for me.
I closed this PR because i want the people that know what they are doing to manage it, but still i'll answer the questions if you are willing to know my opinion.
Probably an new PR with romanian translation updated will be my last on this repo because i want this app in my language too :smiley:
I hope that a newer version will be in play store soon and good luck with it.

**Pull request closed without merging**